lib/topology/logging.py

```diff
@@ -0,0 +1,158 @@
+#
+# Copyright (C) 2015-2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Logging module for the Topology Modular Framework.
+"""
+
+from __future__ import unicode_literals, absolute_import
+from __future__ import print_function, division
+
+import logging
+from os.path import join
+from distutils.dir_util import mkpath
+
+
+class BaseLogger(object):
+    def __init__(self, name, level=logging.NOTSET, log_dir=None):
+        self._name = name
+        self._level = level
+        self._log_dir = log_dir
+
+        self.logger = logging.getLogger(name)
+        self.logger.propagate = False
```

Comment:
Can we remove this so that a root handler is able to grab all level X messages include the ones generated by this logging framework?